### ISSUE TITLE
Publishing history for languages

### ISSUE THREAD

**Issue by bicolino34** [internetarchive/openlibrary#6245]
We have pages for languages https://openlibrary.org/languages and it would be cool to have a publishing history graph for languages like the one subjects have.

**Comment by tuminzee** [comment]
@jimchamp what do you think about this issue?
should we post this in the slack channel, and check if anybody is willing to contribute?

**Comment by jimchamp** [comment]
I don't think that Slack is the correct place to solicit contributions from folks.  If anybody reading this would like to propose a solution, I'd be happy to give feedback.
